.github/workflows/CICD.yml

```diff
@@ -18,11 +18,8 @@ jobs:
     name: Ensure 'cargo fmt' has been run
     runs-on: ubuntu-20.04
     steps:
-    - uses: actions-rs/toolchain@v1
+    - uses: dtolnay/rust-toolchain@stable
```

Comment:
Just as a hint for other reviewers: this action seems to use a "minimal" profile by default: https://github.com/dtolnay/rust-toolchain/issues/41#issuecomment-1291231129